lavalink/Client.py

```diff
@@ -17,8 +17,8 @@ def set_log_level(log_level):
 
 
 class Client:
-    def __init__(self, bot, log_level = logging.INFO, loop = asyncio.get_event_loop(), default_node: int = 0,
-                 player = DefaultPlayer, rest_round_robin: bool = False):
+    def __init__(self, bot, log_level=logging.INFO, loop=asyncio.get_event_loop(), default_node: int = 0,

```

Comment:
Why was this changed? It's no longer consistent with the rest :(

Reply:
I assume that PEP8 want spaces when you specify a type and no spaces when you don't

Reply:
Ah, would it be worth adding types?

Reply:
I don't think so. 

Otherwise we could add an exception to flake8 to avoid error with travis/run_tests.py

Reply:
eh, nevermind it's fine as-is